src/PeriodEnd/Refunds/SFA.DAS.ProviderPayments.Calc.Refunds/Services/Dependencies/ILearnerBuilder.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+using SFA.DAS.ProviderPayments.Calc.Refunds.Dto;
+
+namespace SFA.DAS.ProviderPayments.Calc.Refunds.Services.Dependencies
+{
+    public interface ILearnerBuilder
+    {
+        List<LearnerData> CreateLearnersForThisProvider(long ukprn);
```

Comment:
Is 'This' required? I'm on the fence,

Reply:
Yes, it nicely breaks out the code and allows a bulk SQL read (which is the benefit we advocated on the PaymentsDue code)

Reply:
I meant the 'This' in the name ;)

Reply:
i see, ;-(